web/api/api.go

```diff
@@ -7,22 +7,27 @@ package api
 import (
```

Comment:
At this point, it probably makes sense for this code to be moved into pkg/flow too since there's a logical circular dependency between them.

Reply:
I would rather we work on removing the logical dependency, IMO FlowAPI should be a peer to Flow and not a soft-dependency.